python-sdk/tests/benchmark/Makefile

```diff
@@ -1,4 +1,4 @@
-all: clean setup_gke container run_job
+all: clean setup_gke container run_job wait_for_completion teardown_gke
```

Comment:
The `teardown_gke` step could be dreadful to parallel benchmark jobs triggered from other branches/PRs :see_no_evil: 
It is probably worth adding a separate ticket to address this (perhaps only teardown if no other jobs are running).

Reply:
I have created an issue to read cluster names from env maybe we can consider changing this in that ticket https://github.com/astronomer/astro-sdk/issues/934

Reply:
@pankajastro As per my discussion with @tatiana I think we should not teardown the cluster as of now. We can address it in https://github.com/astronomer/astro-sdk/issues/934

Reply:
I'm ok to disable the teardown for now.